_bibliography/pint.bib

```diff
@@ -951,6 +951,15 @@ @article{MadayEtAl2007
 	year = {2007},
 }
 
+@inbook{Ulbrich2007,
+	author = {Stefan Ulbrich},
+	title = {7. Generalized SQP Methods with ``Parareal'' Time-Domain Decomposition for Time-Dependent PDE-Constrained Optimization},
+	booktitle = {Real-Time PDE-Constrained Optimization},
+	chapter = {},
+	pages = {145-168},
+	doi = {10.1137/1.9780898718935.ch7},
+}
+
```

Comment:
For some reason, this entry does not show up when I test locally. Can you confirm it is correctly generated for you?